docs/deploy-and-review-process.md

```diff
@@ -0,0 +1,59 @@
+# Deploy and review process
+
+This documentation outlines our current processes for how we deploy the latest changes to ethereum.org, and how we prioritize items in our review process.
+
+## Deploy process
+
+Anything merged into the `dev` branch of [ethereum.org](http://ethereum.org) will go live on ethereum.org at the next release. Currently, we deploy the latest changes to [ethereum.org](http://ethereum.org) twice per week.
+
+These are currently:
+
+Monday between 17:00 and 23:00 UTC.
+
+Thursday between 17:00 and 23:00 UTC.
```

Comment:
Do we want to provide explicit times here?

Reply:
Maybe not.. the idea was to avoid people asking why their PR wasn't merged at 06:00 UTC on Monday. Maybe we see if we're having issues without it?

Reply:
I think we can just change the text to say `Currently we deploy on Mondays and Thursdays every week`. I think we can just condense that down to one line